ziti/cmd/ziti/cmd/edge_controller/encryption_var.go

```diff
@@ -0,0 +1,49 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package edge_controller
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+type encryptionVar string
+
+func (f *encryptionVar) String() string {
+	return fmt.Sprint(string(*f))
+}
+
+func (f *encryptionVar) Set(value string) error {
+	value = strings.ToUpper(value)
+	if value != "ON" && value != "OFF" && value != "TRUE" && value != "FALSE" {
+		return errors.New("Invalid option -- must specify either 'ON' or 'OFF'")
+	}
+	*f = encryptionVar(value)
+	return nil
+}
+
+func (f *encryptionVar) Get() bool {
+	if string(*f) == "ON" || string(*f) == "TRUE" {
```

Comment:
personally i think this should normalize the input to upper or lower too... but maybe i'm just overly cautious? :)

Reply:
yes, no reason to require capital letters

Reply:
We do not require caps at the cmd line. The user can enter lower, upper, or mixed case for the on|off|true|false value.  This up-casing is just to make internal comparison easy.